WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -59,6 +59,37 @@ struct PointOfSaleDashboardView: View {
                 }
             }
         })
+        .posModal(isPresented: $viewModel.showExitPOSModal, fixedWidth: false, fixedHeight: false, content: {
+            VStack(spacing: 0 ) {
```

Comment:
Lets move it to a separate method or a file just as we have with `PointOfSaleCardPresentPaymentAlert` to keep this method cleaner.

Reply:
➕